docs/index.asciidoc

```diff
@@ -109,20 +111,87 @@ append and indirect cannot be combined and will fail validation. +
 `%{&+something}` will add a value to the `+something` key, again probably unintended. +
 ===============================
 
-*Delimiter repetition:* +
-In the source text if a field has variable width padded with delimiters, the padding will be ignored. +
-e.g. for texts of:
+===== Multiple Consecutive Delimiter Handling
+
+[IMPORTANT]
+===============================
+Starting from version 1.1.1 of this plugin, multiple found delimiter handling has changed.
+Now multiple consecutive delimiters will be seen as missing fields by default and not padding.
+If you are already using Dissect and your source text has fields padded with extra delimiters,
+you will need to change your config. Please read the section below.
+===============================
+
+====== Empty data between delimiters
+Given this text as the sample used to create a dissection:
+....
+John Smith,Big Oaks,Wood Lane,Hambledown,Canterbury,CB34RY
+....
+The created dissection, with 6 fields, is:
+....
+%{name},%{addr1},%{addr2},%{addr3},%{city},%{zip}
+....
+When a line like this is processed:
 ....
-00000043 ViewReceiver  I
-000000b3 Peer          I
+Jane Doe,4321 Fifth Avenue,,,New York,87432
+....
+Dissect will create an event with empty fields for `addr2 and addr3` like so:
+....
+{
+  "name": "Jane Doe",
+  "addr1": "4321 Fifth Avenue",
+  "addr2": "",
+  "addr3": "",
+  "city": "New York"
+  "zip": "87432"
+}
 ....
-with a dissection of `%{a} %{b} %{c}`; the padding is ignored, `event.get([c]) -> "I"`
 
-[NOTE]
-====
+====== Delimiters used as padding to visually align fields
```

Comment:
I hate to break this to you, but our doc build doesn't currently support level 6 (======) headings, so this gets treated as text. If you want a quick fix, just change this heading (and the one at line 124) to a level 5 heading (=====). Other products (like ES) violate heading levels all over the place to get around asciidoc limitations, so there's no reason why you shouldn't. By the time you get to a level 5 heading, the font size differences are so minute anyhow that no user will notice that you're using the same heading level as the parent section. :-)

Reply:
All good information. 👍